Packs/Incapsula/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Imperva Incapsula
+- %%UPDATE_RN%%
```

Comment:
Please write here your change/addition to the integration (see our release notes [best-practices](https://xsoar.pan.dev/docs/documentation/release-notes#examples-and-best-practices) guide)